### PR TITLE
numpy-sugar

### PR DESCRIPTION
**Pull request by Horta**

### CODE REVIEW

File: recipes/numpy-sugar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+    - build-capi
+    - ncephes
+    - numpy >=1.9.3
```
Comment (jakirkham) [inline recipes/numpy-sugar/meta.yaml]: Does this link to NumPy?

Comment (Horta) [inline recipes/numpy-sugar/meta.yaml]: Nope... why? I mean, I python-import numpy...

**Comment by ocefpaf** [comment]
So... `numpy-sugar` feedstock already exists. Is it a rename case? You can rename the package in the feedstock.

**Comment by Horta** [comment]
Hi @ocefpaf what exist is https://github.com/conda-forge/numpy_sugar-feedstock , with that ugly underscore. I would like to remove that previous feedstock (I cannot rename the repository's name, only the package's name).

**Pull request closed without merging**